sphinx/util/nodes.py

```diff
@@ -132,6 +137,11 @@ def is_translatable(node):
 META_TYPE_NODES = (
     addnodes.meta,
 )
+ALLOWED_TRANSLATED_NODES = (
```

Comment:
What is difference from `translatable` nodes?

On i18n transform, it is described as following:
```
# ignore block markups
if not isinstance(patch, ALLOWED_TRANSLATED_NODES):
    continue  # skip
```
Are these same meaning?

Reply:
Not same.
When `msgstr` in po files contain block markup, `if not isinstance(patch, ALLOWED_TRANSLATED_NODES):` ignore such translations.

Reply:
I feel it is confusable. Could you rename this please to more appropriate name.

Reply:
I thought about defining it by naming rather than listing node classes in the code of `i18n.py`. So, I named `ALLOWED_TRANSLATED_NODES` to them, but this was not a property of being given a single name.
The nodes.title that added this time is a hacky code to handle in pair with the processing in `i18n.py`.
I'll revert this part and add a comment for each node classes.